libs/blocks/modal/modal.css

```diff
@@ -204,6 +204,17 @@
   .dialog-modal.commerce-frame > .fragment > .section {
     height: 100vh;
   }
+
+  .dialog-modal.xl,
+  .dialog-modal.xl > .fragment,
+  .dialog-modal.xl > .fragment > .section {
+    height: 100%;
+  }
+
+  .dialog-modal.xl .milo-iframe {
+    height: 100%;
+    padding-bottom: 0;
+  }
```

Comment:
what are implicit values for those rules otherwise?
i'm noticing same setup for `> 1200px` and  `.dialog-modal.commerce-frame .milo-iframe`

Reply:
@npeltier 
Without these rules:
- for the .dialog-modal the height value is 'fit-content'.
- for .fragment and .section we don't set values, so the default value ('auto') applies.
- for .milo-iframe height is set to 0, and padding-bottom to 56.25%.

The class .commerce-frame should be used only on commerce related modal windows.
'What's Included' content is not related to commerce, so it makes sense to use a more general class name.